src/rosdiscover/interpreter/summary.py

```diff
@@ -8,12 +8,20 @@
 
 @attr.s(frozen=True, slots=True, auto_attribs=True)
 class NodeSummary:
+
```

Comment:
Last nitpick: Can we get rid of this space?

Reply:
Done